src/extension.ts

```diff
@@ -1302,6 +1302,13 @@ function createIdfTerminal() {
     const envSetCmd =
       process.platform === "win32" ? `set "IDF_PATH=` : `export IDF_PATH="`;
     espIdfTerminal.sendText(`${envSetCmd}${modifiedEnv.IDF_PATH}"`);
+    const setPythonEnvCmd =
```

Comment:
Is this used somewhere ?

Reply:
For some reason, we need to override IDF_PATH and PATH using terminal export/set because it seems the terminal API is not overriding variables just post appending.

I just noticed there is an option to strictly use the env we pass to the terminal. Will update now.